specification/app/resource-manager/Microsoft.App/stable/2025-01-01/CommonDefinitions.json

```diff
@@ -899,6 +900,32 @@
         }
       }
     },
+    "NfsAzureFileProperties": {
+      "description": "NFS Azure File Properties.",
+      "type": "object",
+      "properties": {
+        "server": {
```

Comment:
>server [](http://example.com/codeflow?start=9&length=6)

Is this just a host name, an Azure resource ID, a guid, something else? Please add more to the description to help know what this property contains. If it's an Azure resourceId, include that in the name (e.g. serverResourceId). If it's a server host name, consider something like serverName or serverHostName.

Reply:
Add more description for this. @mentat9, This is a feature that has been enabled in preview version which is named as "server", the name also same as how AKS cluster enable nfs azure file share. we want to keep all these naming aligns. refer
https://learn.microsoft.com/en-us/azure/aks/azure-files-csi#use-a-persistent-volume-with-private-azure-files-storage-private-endpoint